ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -388,20 +394,25 @@ void RSkComponentImage::requestNetworkImageData(ImageSource source) {
     auto responseCacheControlData = responseData->headerBuffer.find("Cache-Control");
     if(responseCacheControlData != responseData->headerBuffer.items().end()) {
       std::string responseCacheControlString = responseCacheControlData->second.asString();
-      canCacheData_ = shouldCacheData(responseCacheControlString);
+      canCacheData_ = remoteCurlRequest->shouldCacheData();
```

Comment:
we dont need getCacheMaxAgeDuration as well right ? since CUrlResponse stores the this value in cacheExpiryTime . can the same be used here